packages/frontend2/src/app/_components/nav/NavSideBarWrapper.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import { cn } from '~/utils/cn'
+import { useMobileNav } from './MobileNavContext'
+
+export const NavSideBarWrapper = ({
+  children,
+  legacyNav,
+}: { children: React.ReactNode; legacyNav?: boolean }) => {
+  const { open } = useMobileNav()
+  const [resizing, setResizing] = useState(false)
+
+  const sharedSizeClasses = cn(
+    'w-full xl:w-[240px] 2xl:w-[280px] h-screen h-[100dvh]',
```

Comment:
'h-[100dvh]' applies the same CSS properties as 'h-screen'

Reply:
It actually doesn't 😛 But I just found out there's `h-dvh` which does the same thing!